Library/Homebrew/formula_installer.rb

```diff
@@ -942,13 +943,25 @@ def post_install
     @show_summary_heading = true
   end
 
-  def pour
+  def fetch_and_build
+    compute_dependencies unless ignore_deps?
+    pour_bottle? ? fetch_bottle : build
+
+    @fetched = true
+  end
+
+  def fetch_bottle
     if (bottle_path = formula.local_bottle_path)
       downloader = LocalBottleDownloadStrategy.new(bottle_path)
     else
       downloader = formula.bottle
-      downloader.fetch
+      downloader.fetch unless @prefetched
```

Comment:
This variable was renamed it seems:
```suggestion
      downloader.fetch unless @fetched
```